### ISSUE TITLE
builder not returning all releases for a version

### ISSUE THREAD

**Issue by mwrock** [habitat-sh/builder#1615]
This was noticed by a customer who has a package version with 132 releases. It can also be observed for `core/7zip`. Notice that `https://bldr.habitat.sh/v1/depot/pkgs/core/7zip/16.02` returns 24 releases:

```
{
range_start: 0,
range_end: 23,
total_count: 24,
data: [
{
```
However, the UI states that the total is 30:
![image](https://user-images.githubusercontent.com/655165/131176713-23bd9416-21ef-43bf-bd72-e76b4d37b5df.png)

Quwrying the DB `select ident from origin_packages where origin='core' and name='7zip' order by ident;` also returns 30 packages. So there are indeed 30 but the 6 oldest packages are missing from the UI and the rest api.

**Comment by y-me-y** [comment]
Just as a side note this query returns 30 packages in the results https://bldr.habitat.sh/v1/depot/channels/core/unstable/pkgs/7zip/16.02

**Comment by sajjaphani** [comment]
For version 16.04, on clicking the Linux node also shows the same Windows packages.
<img width="1127" alt="Screenshot 2021-09-23 at 3 54 35 PM" src="https://user-images.githubusercontent.com/2322779/134491723-fbd17751-c798-4ac9-a60e-700a4c24932a.png">

**Comment by mwrock** [comment]
fixed by #1627
